storage/src/journal/variable.rs

```diff
@@ -59,16 +59,6 @@
 //! some in-memory state. `Journal` is heavily optimized for this pattern and provides a `replay`
 //! method that iterates over multiple `sections` concurrently in a single stream.
 //!
-//! ## Skip Reads
```

Comment:
Now that the journal handles compression, it isn't safe to do these reads.

I think that's a fair trade given this was pretty complex to do right (and required layers of checksums).